lightning_pose/utils/fiftyone.py

```diff
@@ -0,0 +1,511 @@
+import fiftyone as fo
```

Comment:
small ocd comment: to keep imports organized i usually do the following: import outside functions/modules first, in alphabetical order, skip a line, then import package-level functions/modules next, also in alphabetical order:

```python
import fiftyone as fo
import numpy as np
from omegaconf import DictConfig, OmegaConf, ListConfig
import os
import pandas as pd
from tqdm import tqdm
from typeguard import typechecked
from typing import Dict, List, Optional, Union, Callable

from lightning_pose.utils.io import return_absolute_path, return_absolute_data_paths
from lightning_pose.utils.plotting_utils import get_videos_in_dir

```

not recommending we do this for every file right now, but would be good to keep this in mind when importing new things to slow the increase of entropy

Reply:
ok -- that makes sense. wasn't aware of this method; will start doing that from now on.